docs/elements/examples/shadcn-ui.mdx

```diff
@@ -0,0 +1,505 @@
+---
+description: Learn how to compose Clerk Elements with shadcn/ui to build custom sign in and sign up flows.
+---
+
+# shadcn/ui
+
+The following examples demonstrate how to compose Clerk Elements with shadcn/ui to build custom sign-in and sign-up flows.
+## Prerequisites
+
+To use these examples, you must first:
+
+- Complete the [shadcn/ui Next.js installation guide](https://ui.shadcn.com/docs/installation/next)
+- Install the [Button](https://ui.shadcn.com/docs/components/button), [Card](https://ui.shadcn.com/docs/components/card), [Input](https://ui.shadcn.com/docs/components/input), and [Label](https://ui.shadcn.com/docs/components/label) components within your project
```

Comment:
```suggestion
- Install the [Button](https://ui.shadcn.com/docs/components/button), [Card](https://ui.shadcn.com/docs/components/card), [Input](https://ui.shadcn.com/docs/components/input), and [Label](https://ui.shadcn.com/docs/components/label) components within your project
	```shell
	npx shadcn-ui@latest add button card input label
	```
```
Adds the actual command they can run